gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-client/src/main/java/org/apache/gobblin/service/FlowConfigClient.java

```diff
@@ -86,17 +90,17 @@ public FlowConfigClient(RestClient restClient) {
    * @param flowConfig flow configuration attributes
    * @throws RemoteInvocationException
    */
-  public void createFlowConfig(FlowConfig flowConfig)
+  public String createFlowConfig(FlowConfig flowConfig)
       throws RemoteInvocationException {
     LOG.debug("createFlowConfig with groupName " + flowConfig.getId().getFlowGroup() + " flowName " +
         flowConfig.getId().getFlowName());
 
-    CreateIdRequest<ComplexResourceKey<FlowId, EmptyRecord>, FlowConfig> request =
+    CreateIdRequest<ComplexResourceKey<FlowId, FlowStatusId>, FlowConfig> request =
         _flowconfigsRequestBuilders.create().input(flowConfig).build();
-    ResponseFuture<IdResponse<ComplexResourceKey<FlowId, EmptyRecord>>> flowConfigResponseFuture =
+    ResponseFuture<IdResponse<ComplexResourceKey<FlowId, FlowStatusId>>> flowConfigResponseFuture =
         _restClient.get().sendRequest(request);
 
-    flowConfigResponseFuture.getResponse();
+    return parseFlowExecutionId(flowConfigResponseFuture.getResponse());
```

Comment:
Why not return a FlowStatusId instead of a String?

Reply:
The only extra fields in FlowStatusId are flow name and group; which user is providing.
Does it call for parsing a header and recreate a FlowStatusId?